packages/legend-application-query/src/stores/LegendQueryApplicationPlugin.tsx

```diff
@@ -63,6 +65,10 @@ export type NewQueryNavigationPath = (
   editorStore: ExistingQueryEditorStore,
 ) => string | undefined;
 
+export type QueryGraphBuilderGetter = (
```

Comment:
where is this used ?

Reply:
This is used in https://github.com/finos/legend-studio/pull/3488/files#diff-5dae4710ada9d402191df7256d5cc7d4502e3b40a8f1b9acf960143f7ee6adfeR690
This is for loading an empty function to aovid `buildFullGraph` for ExistingQueryEditorStore with dataspace context only